src/priorityq.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding utf-8 -*-
+
+
+class Priorityq(object):
+
+    def __init__(self, iterable=None):
+        '''Initialise a Priorityq from a list of lists or tuples
+            where the zeroeth element is the priority'''
+        if iterable is None:
+            self.priorityq = None
+        else:
+            try:
+                self.priorityq = sorted(iterable, key=lambda tup: tup[0])
```

Comment:
Another good recipe for this involves using importing `operator.itemgetter`:

`newlist = sorted(iterable, key=itemgetter(0))`
A little more succinct.

Sorting the initial input list will work just fine at a time complexity of O(n); subclassing your Heap will allow sorting at O(logn).